anaconda

```diff
@@ -1139,7 +1139,7 @@ if __name__ == "__main__":
             iutil.setenv("ftp_proxy", proxy.url)
             iutil.setenv("HTTPS_PROXY", proxy.url)
 
-    if flags.noverifyssl:
+    if flags.noverifyssl and ksdata.method.method == "url":
         ksdata.method.noverifyssl = flags.noverifyssl
```

Comment:
We should probably check `hasattr(ksdata.method, "noverifyssl")`, because there can be several methods with this attribute (for example liveimg).

Reply:
You are right, fixed, thank you.